rpm/Makefile

```diff
@@ -0,0 +1,10 @@
+rpm:
```

Comment:
Should have a `.PHONY rpm` otherwise make will think this target is producing an output file called `rpm`.